Paco-Server/ear/default/WEB-INF/web.xml

```diff
@@ -36,6 +36,33 @@
   </servlet-mapping>
   
   <servlet>
+    <servlet-name>ServerInsertServlet</servlet-name>
+    <servlet-class>com.google.sampling.experiential.server.CloudSqlInsertServlet</servlet-class>
+  </servlet>
+  <servlet-mapping>
+    <servlet-name>ServerInsertServlet</servlet-name>
+    <url-pattern>/csInsert</url-pattern>
+  </servlet-mapping>
+  
+  <servlet>
+    <servlet-name>ServerCreateServlet</servlet-name>
+    <servlet-class>com.google.sampling.experiential.server.CloudSQLCreateServlet</servlet-class>
+  </servlet>
+  <servlet-mapping>
+    <servlet-name>ServerCreateServlet</servlet-name>
+    <url-pattern>/csCreate</url-pattern>
+  </servlet-mapping>
+  
+  <servlet>
+    <servlet-name>ServerSearchServlet</servlet-name>
+    <servlet-class>com.google.sampling.experiential.server.CloudSqlSearchServlet</servlet-class>
+  </servlet>
+  <servlet-mapping>
+    <servlet-name>ServerSearchServlet</servlet-name>
+    <url-pattern>/csSearch</url-pattern>
```

Comment:
This is a client end point. So, it does not fall into admin role. Also I moved this URL to the non admin resource collection group